pandas/io/parsers.py

```diff
@@ -825,6 +825,22 @@ def _validate_usecols_arg(usecols):
     return usecols
 
 
+def _validate_parse_dates_arg(parse_dates):
```

Comment:
On a separate note, to what extent should this validation go?  Is it just type checking?  Or are we also validating element types?  Also, are mixed types allowed?  I presume a mixture of `int` and `string` is not, but what about `int` and `list` for example?